experiments/saxpy/openmp/execute_experiment.tpl

```diff
@@ -4,9 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-{batch_nodes}
-{batch_ranks}
-{batch_timeout}
+{allocation_directives}
```

Comment:
This PR can now generate appropriate headers and invocations, so the only thing a system needs to specify is a `scheduler` variable, at which point the modifier will set:

* {allocation_directives}
* {mpi_command}
* {batch_submit}